app/controllers/review.controller.js

```diff
@@ -0,0 +1,163 @@
+const db = require("../models");
+const Review = db.review;
+
+exports.getAll = async (req, res) => {
+  const resumeId = req.params.resumeId;
+
+  await Review.findAll({
+    where: {
+      resumeId: resumeId,
+    },
+  })
+    .then((data) => {
+      res.send(data);
+    })
+    .catch((err) => {
+      res.status(500).send({
+        message:
+          err.message ||
+          `Something went wrong while trying to fetch all reviews for resume with id ${resumeId}`,
+      });
+    });
+};
+
+exports.getForId = async (req, res) => {
+  const id = req.params.id;
+
+  await Review.findOne({
+    where: {
+      id: id,
+    },
+  })
+    .then((data) => {
+      res.send(data);
+    })
+    .catch((err) => {
+      res.status(500).send({
+        message:
+          err.message ||
+          `Something went wrong while trying to fetch all reviews for review ${id}`,
+      });
+    });
+};
+
+exports.create = async (req, res) => {
+  const resumeId = req.params.resumeId;
+
+  const validation = validateReview(req.body);
+
+  if (!validation.valid) {
+    // Return an error message if validation fails
+    return res.status(400).json({
+      message: "Validation error",
+      details: validation.errors,
+    });
+  }
```

Comment:
When creating a review, the summary is almost 100% of the time going to be empty, and if a status is not provided, it will always be in-progress, so there's no need to validate for null, just provide a default value

Reply:
Fixed